charts/nfs-subdir-external-provisioner/README.md

```diff
@@ -61,6 +61,7 @@ The following tables lists the configurable parameters of this chart and their d
 | `storageClass.reclaimPolicy`        | Method used to reclaim an obsoleted volume                  | `Delete`                                          |
 | `storageClass.provisionerName`      | Name of the provisionerName                                 | null                                              |
 | `storageClass.archiveOnDelete`      | Archive pvc when deleting                                   | `true`                                            |
+| `storageClass.onDelete`             | Defines strategy on pvc deletion, overrides archiveOnDelete | null                                              |
```

Comment:
can the description also include the valid set of values, please.

Reply:
There is a table explaining onDelete and its relation with archiveOnDelete on the main readme. I think it would be confusing/hard to maintain to have it in multiple places, what do you think of adding a reference/link to the table on the root readme?

Reply:
the table in the readme contains detailed explanation about the values. here it is required to present in practice what values this variable can receive without too much details. not a duplication imo.